helm/rook-operator/charts/monitoring/templates/ceph-pool-rules.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.global.rookEnabled }}
+{{- if .Values.global.isManagementCluster }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: {{ .Values.nameStub }}-ceph-pool.rules
+  namespace: {{ .Values.rules.deploymentNamespace }}
+spec:
+  groups:
+  - name: ceph-pool.rules
+    rules:
+    - alert: CephPoolQuotaBytesNearExhaustion
+      annotations:
+        description: '{{`Storage pool {{ $labels.name }} quota usage has crossed 70%.`}}'
+        opsrecipe: rook-ceph/
+      expr: |
+        (ceph_pool_stored_raw * on (pool_id) group_left(name)ceph_pool_metadata) / ((ceph_pool_quota_bytes * on (pool_id) group_left(name)ceph_pool_metadata) > 0) > 0.70
+      for: 1m
+      labels:
+        area: kaas
+        severity: notify
+        team: rocket
+        topic: managementcluster
+    - alert: CephPoolQuotaBytesCriticallyExhausted
+      annotations:
+        description: '{{`Storage pool {{ $labels.name }} quota usage has crossed 90%.`}}'
+        opsrecipe: rook-ceph/
+      expr: |
+        (ceph_pool_stored_raw * on (pool_id) group_left(name)ceph_pool_metadata) / ((ceph_pool_quota_bytes * on (pool_id) group_left(name)ceph_pool_metadata) > 0) > 0.90
+      for: 1m
+      labels:
+        area: kaas
+        severity: notify
+        team: rocket
+        topic: managementcluster
```

Comment:
Pool quota isn't necessarily the same as available storage so it makes sense to alert on this